src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1546,19 +1560,24 @@ void load_from_sv6(const char* path)
     }
     catch (const ObjectLoadException&)
     {
-        gErrorType = ERROR_TYPE_FILE_LOAD;
-        gErrorStringId = STR_FILE_CONTAINS_INVALID_DATA;
+        uint8_t gErrorType = ERROR_TYPE_FILE_LOAD;
+        rct_string_id gErrorStringId = STR_FILE_CONTAINS_INVALID_DATA;
+        show_error(gErrorType, gErrorStringId);
```

Comment:
Why not pass the directly the value to the parameter? You don't have to assign variables first.

Reply:
Fixed it in a new commit